edkahara/iReporter
# [finishes #162152682] create a home page

__What does this PR do?__
Create a home page.

__Description of tasks to be completed.__
Created a home page with instructions on how to use iReporter.

__How should it be manually tested?__
Clone the project. Check out the ft-home-page-#162152682 branch. Open the ui folder and run the index.html file using any browser.

__Relevant pivotal tracker stories.__
#162152682

__Relevant screenshots.__

![home page](https://user-images.githubusercontent.com/20911240/48897388-01892580-ee5b-11e8-80ff-898291af2a5e.png)
